PyRedPitaya/raw_memory.py

```diff
@@ -3,11 +3,25 @@
 from ctypes import *
```

Comment:
Star imports make linting pretty hard and less useful. This should also be changed in the rest of the package.

Reply:
Fixed by https://github.com/linien-org/pyrp3/pull/2/commits/bd379947daaa73a7b5bee9947fc51ccb58469f20